lib/factory.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {ALGORITHM} from './constants.js';
+import {webcrypto, EcdsaCurve, EcdsaHash} from './ecdsa.js';
+
+export function createSigner({id, secretKey}) {
+  if(!secretKey) {
+    throw new Error('A secret key is not available for signing.');
```

Comment:
Not sure about the phrasing here and in verify call below.  Maybe it should be more direct such as "secretKey required for signing."?   I'm not sure what we use in other libraries that are not just using an assert library.  Also could be `TypeError`s.

Reply:
This error message was used in the `ed25519-multikey` library, so I decided to add it here for consistency. @gannan08 thoughts?

Reply:
For now, I will update these to more actionable error messages.

Reply:
@kezike +1. That was going to be my recommendation.